cli/src/main/java/de/jplag/cli/CLI.java

```diff
@@ -1,7 +1,6 @@
 package de.jplag.cli;
 
-import static picocli.CommandLine.Model.UsageMessageSpec.SECTION_KEY_FOOTER;
-import static picocli.CommandLine.Model.UsageMessageSpec.SECTION_KEY_OPTION_LIST;
+import static picocli.CommandLine.Model.UsageMessageSpec.*;
```

Comment:
Can you disable the auto star importing for your IDE, will just lead to more noise in the changes.